### DISCUSSION TITLE
TrustedHTML error

### DISCUSSION THREAD

**Discussion by MrSakine** [webdriverio/webdriverio#10141] [Q&A]
Hello,
I'm trying to get an HTMLELEMENT via document.getElementById(...) and got this error:
Evaluation failed: TypeError: Failed to set the 'innerHTML' property on 'Element': This document requires 'TrustedHTML' assignment.

**Comment by christian-bromann** [comment]
@MrSakine not sure what you are trying to achieve here but if you want to fetch elements based on an id selector, use something like `$('#someId')`. For further information, see the selector docs: https://webdriver.io/docs/selectors
